test/unit_tests/braket/default_simulator/cirq/test_openqasm_conversion.py

```diff
@@ -0,0 +1,216 @@
+import cirq
+import pytest
+
+from braket.default_simulator.cirq.program_context import ProgramContext
+from braket.default_simulator.openqasm.interpreter import Interpreter
+
+
+@pytest.mark.parametrize(
+    "openqasm",
+    (
+        (
+            """
```

Comment:
This is the correct approach. The default simulator should not use the BDK since it is a dependency of the BDK and that would create a circular import.